safe-global/web-core
# feat: execution method selector

## What it solves

Resolves #1894

## How this PR fixes it

The option to choose the execution method is now always present, regardless of remaining relays.

## How to test it

- Ensure relaying is possible.
- Ensure it's possible to execute transactions via the connected wallet (when there are remaining relays).
- Ensure it's possible to execute transactions via the connected wallet (when there are _no_ remaining relays). There should be no option to select the execution method.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/234816741-1c10472c-4553-4186-8bbc-2947cd3e5190.png)
Note: the duplicate "Goerli" is there because we have no icon on Goerli.

![image](https://user-images.githubusercontent.com/20442784/234816818-7542a807-1dfe-4d9e-89c2-dd55fabe8734.png)

![image](https://user-images.githubusercontent.com/20442784/234816843-d0eeac52-f71e-49a5-8d4f-f60c0c619227.png)

## Checklist
